src/Shopsys/ShopBundle/Controller/Front/ScriptController.php

```diff
@@ -48,6 +48,7 @@ public function embedAllPagesGoogleAnalyticsScriptAction()
 
         return $this->render('@ShopsysShop/Front/Inline/MeasuringScript/googleAnalytics.html.twig', [
             'trackingId' => $this->scriptFacade->getGoogleAnalyticsTrackingId($this->domain->getId()),
+            'domainId' => $this->domain->getId(),
```

Comment:
this is connected to my other comment - if GA is not activated, then GTM scripts won't be rendered :confused: (see the `if` above)

Reply:
You are right, fixed